src/metabase/util/honeysql_extensions.clj

```diff
@@ -257,6 +257,17 @@
       expr
       (cast sql-type expr)))
 
+(defn cast-unless-type-in
```

Comment:
Add `:added` metadata since new public symbol?

Reply:
probably a good idea. I'll add it

Reply:
added